cli/module_registry.go

```diff
@@ -209,21 +210,19 @@ func UploadModuleAction(c *cli.Context) error {
 				"if you want to upload a version without a meta.json, you must supply a module name and namespace (or module name and org-id)",
 			)
 		}
-		moduleID, err = updateManifestModuleIDWithArgs(c, client, nameArg, publicNamespaceArg, orgIDArg)
-		if err != nil {
-			return err
-		}
+		// get from the BE?
```

Comment:
We still need it on upload because a `meta.json` is not required for an upload. There is no `meta.json`, the user can provide a namespace or an orgid + name